lib/streamlit/secrets.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018-2021 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import threading
+from typing import Any, Optional, Mapping, MutableMapping
+
+import toml
+
+import streamlit as st
+
+_SECRETS_LOCATION = os.path.expanduser(os.path.join(".", ".streamlit", "secrets.toml"))
```

Comment:
I believe the `os.path.expanduser` ends up being a no-op here unless the intention was to look for secrets in `~/.streamlit/secrets.toml`, in which case there's a `"~"` arg missing from `os.path.join`.

Reply:
No, you're right - it's supposed to be `./streamlit/secrets.toml`. Thanks for the catch